documentation/onboard-service.md

```diff
@@ -109,9 +111,14 @@ Take note of the name servers of the newly created DNS zone and request delegati
 - [Infrastructure and networks](https://technical-guidance.education.gov.uk/infrastructure/hosting/dns/#service-now) for the education.gov.uk zone
 - [GDS](https://technical-guidance.education.gov.uk/infrastructure/hosting/dns/#official-service-gov-uk-domains) for the service.gov.uk zone
 
+### Create domain.sh in global config
+In global config, create domain.sh and also update the Makefile have a make command for it
+
 ### Create custom domain per environment
 This configures the custom domain for a particular environment in the previously create DNS zone and front door.
 
+Refer to the new service template in templates/new_service/terraform/domains/environment_domains, for the files needed before running the commands below:
```

Comment:
At this point, users will have already used the template script as per https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/onboard-service.md#generate-code
So there is no template involved anymore.

Reply:
Same as above. Should I remove or use something like "Ensure ..."?